numerics_paper/version2/numerical_chj.tex

```diff
@@ -473,12 +484,19 @@ \section{An entropy-based blending of Rusanov and Roe}\label{sec:blended_rs}
 
 \subsection{The entropy residual}
 Let $\entvar(\bfq)$ and $\entflux(\bfq)$ denote the entropy variable and entropy flux.
+Recall that $Q=[h,hu,hv]^t$. We use the total energy as entropy; that is, we consider 
```

Comment:
I think this should be "^T".

Reply:
Also this should be "q", not "Q".  "Q" is for the cell average.

Reply:
This is a good point. Indeed, I think we need to change other things in that section. I changed Q for q when we use it inside the integrals. Since these integrals are element-based, having Q instead of q means we could simply pull the integrands out. In the formula for Ri, we use eta'(Q_i) so we pull that term out.